tests/test_components/test_simulation.py

```diff
@@ -1538,6 +1538,8 @@ def test_sim_volumetric_structures():
         LARGE_NUMBER * thickness / grid_dl,
         rtol=RTOL,
     )
+    # check that plotting 2d material with alpha<1 doesn't raise an error
+    _ = sim.plot_eps(ax=AX, x=0, alpha=0.2)
```

Comment:
Should we also add a `sim_data.plot_field()` test, since that's how the problem originally came to light?

Reply:
good point, added the test